lib/minitest/mock.rb

```diff
@@ -68,15 +68,15 @@ def initialize(delegator = nil) # :nodoc:
     #   @mock.uses_one_string("bar") # => true
     #   @mock.verify  # => raises MockExpectationError
 
-    def expect(name, retval, args = [], &blk)
+    def expect(name, retval, args = [], times: nil, &blk)
```

Comment:
Since minitest supports ruby 1.8.7 -> 2.2.* - we can't use keyword arguments. Thoughts on turning this into an options hash?